public/app/whiteboards/board/whiteboardsBoardDirective.js

```diff
@@ -1794,9 +1794,15 @@
           scope.whiteboard = $scope.whiteboard;
           scope.closeModal = function(updatedWhiteboard) {
             if (updatedWhiteboard) {
-              $scope.whiteboard = updatedWhiteboard;
-              // Set the title of the window to the new title of the whiteboard
-              $rootScope.header = $scope.whiteboard.title;
+              if (updatedWhiteboard.notFound) {
+                // If an edit has removed the user's access, close and refresh the main whiteboard list
+                $window.close();
+                $window.opener.refreshWhiteboardList();
```

Comment:
Could be, thanks, @nicolaasmatthijs. Reordered, and added a null check on the opener in case it's been closed in the meantime.